client/src/modules/report-group/index.html

```diff
@@ -0,0 +1,170 @@
+        autocomplete="off"
+        required>
+
+        <div class="help-block"
+        ng-messages="ReportGroupRegistrationForm.code.$error"
+        ng-show="ReportGroupRegistrationForm.$submitted">
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+        </div>
+      </div>
+      </div>
+      <!-- name -->
+      <div
+      class="form-group"
+      ng-class="{'has-error' : ReportGroupRegistrationForm.name.$invalid && ReportGroupRegistrationForm.$submitted}">
+
+      <label class="control-label ng-binding">
+        Name
+      </label>
+
+      <div class="input-group col-lg-12">
+        <input class="form-control"
+        type="text"
+        name="name"
+        ng-model="ReportGroupCtrl.reportGroup.name"
+        autocomplete="off"
+        required/>
+        <div class="help-block"
+        ng-messages="ReportGroupRegistrationForm.name.$error"
+        ng-show="ReportGroupRegistrationForm.$submitted">
+
+        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+        </div>
+      </div>
+      </div>
+      <!-- description -->
+
+      <div
+      class="form-group"
+      ng-class="{'has-error' : ReportGroupRegistrationForm.description.$invalid && ReportGroupRegistrationForm.$submitted}">
+
+      <label class="control-label ng-binding">
+        Description
+      </label>
+
+      <div class="input-group col-lg-12">
+        <textarea
+        class="form-control"
+        name="description"
+        ng-model="ReportGroupCtrl.reportGroup.description"j
+        autocomplete="off"
+        required>
+        </textarea>
+        <div class="help-block"
+        ng-messages="ReportGroupRegistrationForm.description.$error"
+        ng-show="ReportGroupRegistrationForm.$submitted">
+          <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+        </div>
+      </div>
+      </div>
+
+
+      <bh-loading-button
+      ng-show="ReportGroupCtrl.selectedReportGroup.selected == false"
+            class="pull-right" loading-state="BedRegistrationForm.$loading">
+              {{ "Save"}}
+          </bh-loading-button>
+
+          <span
+            class="btn btn-danger"
+            ng-click="ReportGroupCtrl.remove(ReportGroupRegistrationForm)"
+            ng-show="ReportGroupCtrl.selectedReportGroup.selected == true"
+            class="pull-right">
+              {{ "Delete"}}
+          </span>
+
+          <bh-loading-button
+            ng-show="ReportGroupCtrl.selectedReportGroup.selected == true"
+            class="pull-right"
+            loading-state="BedRegistrationForm.$loading">
+            {{ "Update"}}
+          </bh-loading-button>
+
+        </form>
+
+      </div>
+      <!-- end new Reporting group form -->
+
+      <!-- start grid of Reports group -->
+
+      <div class="col-lg-7">
+        <!--the grid -->
+
+        <div
+          id="report-groups-grid"
+          ui-grid-auto-resize
```

Comment:
Do you need all these options on the `uiGrid`?!  Remember that every option you add will slow the module down!